assets/icons.tsx

```diff
@@ -685,6 +685,32 @@ export const SearchClockIcon = () => {
   );
 };
 
+export const UserIcon = () => {
+  return (
+    <svg
+      xmlns='http://www.w3.org/2000/svg'
+      width='24'
+      height='24'
+      viewBox='0 0 24 24'
+      fill='none'>
+      <circle cx='12' cy='12' r='9' fill='white' stroke='#222222' />
+      <circle
+        cx='12'
+        cy='10'
+        r='3'
+        fill='white'
+        stroke='#222222'
+        stroke-linecap='round'
```

Comment:
Convert `stroke-linecap` this type of attributes to `strokeLinecap` because a dash into attribute name is not valid in react, it will produce warning while building.